ersilia/hub/fetch/fetch.py

```diff
@@ -1,3 +1,6 @@
+
+
+
```

Comment:
Please we can remove all this added space.

Reply:
I'm on it. Thanks for pointing it out @DhanshreeA

Reply:
@DhanshreeA done

Reply:
@DhanshreeA Can you assign me some other task as well which can be exciting to do